### PR TITLE
UGC-4587 | Update VE loading speed

### PR DESCRIPTION
**Pull request by MrVanosh**

### CODE REVIEW

File: modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js [hunk 1]
```diff
@@ -986,6 +986,12 @@
 			} else if ( pageCanLoadEditor ) {
 				// Allow instant switching to edit mode, without refresh
 				$caVeEdit.off( '.ve-target' ).on( 'click.ve-target', init.onEditTabClick.bind( init, 'visual' ) );
+				$caVeEdit.on( 'mouseover.ve-target', function () {
```
Comment (mszabo-wikia) [inline modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js:992]: Let's demark our changes to this file with `// Fandom change` comments for easier reconciliation later

File: modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js [hunk 2]
```diff
@@ -911,9 +911,9 @@
 				pTabsId = isMinerva ? 'page-actions' :
 					$( '#p-views' ).length ? 'p-views' : 'p-cactions',
 				// Minerva puts the '#ca-...' ids on <a> nodes
-				$caSource = $( 'li#ca-viewsource' ),
-				$caEdit = $( 'li#ca-edit, li#page-actions-edit' ),
-				$caVeEdit = $( 'li#ca-ve-edit' ),
+				$caSource = $( '#ca-viewsource' ),
+				$caEdit = $( '#ca-edit, #page-actions-edit' ),
+				$caVeEdit = $( '#ca-ve-edit' ),
```
Comment (mszabo-wikia) [inline modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js:917]: Why the selector changes?

Comment (MrVanosh) [reply modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js:917]: In our current setup, this code for multi tab is executed so we need to match selectors for our skin. We could change config to make VE know we're not have multi tabs like on standard mediawiki skin but it introduces problems e.g. with section edits as we're have everything prepared for current config.

File: modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js (hunk 1 above)
Comment (MrVanosh) [reply modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js:992]: Added

File: modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js (hunk 2 above)
Comment (mszabo-wikia) [reply modules/ve-mw/preinit/ve.init.mw.DesktopArticleTarget.init.js:917]: If it ultimately makes things simpler, I think we should do it. If it's too large an effort to be done within the scope of this ticket, we could create a follow-up task for it.

**Review by mszabo-wikia — APPROVED** [review]

**Comment by kasperekt** [comment]
@MrVanosh I guess we can close this PR, right?

**Pull request closed without merging**